docs/antrea-network-policy.md

```diff
@@ -100,10 +102,10 @@ Tiers have the following characteristics:
 
 Antrea release 0.9.x introduced support for 5 static tiers. These static tiers
```

Comment:
Is there a specific reason for using "tiers" instead of "Tiers" here and in the section title? Looks like all the others are replaced.

Reply:
This intricate capitalization scheme is to indicate that before 0.9.x "tier" is merely a field or a concept for the Antrea-native policies, while since 1.0 it became a K8s resource, hence capitalized.

Reply:
Understood👌